### PR TITLE
Review coverage testing

### PR DESCRIPTION
**Pull request by sheager**

### CODE REVIEW

**Comment by SheaPrime** [comment]
sheaprime comment

File: Fork Test [hunk 1]
```diff
@@ -3,32 +3,33 @@ ed
 2
 3
 4
-5 lines new commit
-1
-2
```
Comment (SheaPrime) [inline Fork Test]: 1/6

File: Fork Test [hunk 2]
```diff
@@ -3,32 +3,33 @@ ed
...
-3
-4
-5
-d
```
Comment (SheaPrime) [inline Fork Test]: 2/6

File: Fork Test (hunk 1 above)
Comment (sheager) [reply Fork Test]: ya I see that

File: Fork Test [hunk 3]
```diff
@@ -3,32 +3,33 @@ ed
...
-d
 d
 d
 d
 
-6
-7
```
Comment (sheager) [inline Fork Test]: 3/6 should not count it's from submitter.

**Pull request merged**